tests/integration_tests/datasources/test_ec2_ipv6.py

```diff
@@ -41,3 +41,11 @@ def test_dual_stack(client: IntegrationInstance):
     # Block IPv6 requests
     assert client.execute("ip6tables -I OUTPUT -d fd00:ec2::254 -j REJECT").ok
     _test_crawl(client, "http://169.254.169.254")
+
+    # Force NoDHCPLeaseError (by removing dhclient) and assert ipv6 still works
+    # Destructive test goes last
+    client.execute("rm /usr/sbin/dhclient").ok
```

Comment:
```suggestion
    assert client.execute("rm /usr/sbin/dhclient").ok
```

I guess we want to make sure that the test precondition was correctly executed here.